<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliceOrientationWidget</class>
 <widget class="QWidget" name="SliceOrientationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slice Orientation Controller</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="axialRadioButton">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>axial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sagittalRadioButton">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>sagittal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="coronalRadioButton">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>coronal</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
